astropy/nddata/compat.py

```diff
@@ -117,8 +117,8 @@ def uncertainty(self, value):
                     # Raise an error if uncertainty has unit and data does not
                     raise ValueError(
                         "Cannot assign an uncertainty with unit "
-                        "to {} without "
-                        "a unit".format(class_name)
+                        f"to {class_name} without "
```

Comment:
Weird that this has such short lines. I thought black took care of that?

Reply:
I don't think black merges strings except on experimental mode.